grpc/estimator_service.cc

```diff
@@ -12,10 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <cstring>
```

Comment:
This change to the header order will upset the internal linter.  Can you please undo it, and perhaps more deeply we should figure out how to alter the style spec on your side so this doesn't happen again...

Reply:
Resolving so I can merge, but I'll have to manually undo this internally, so we should find a way to make sure it doesn't happen routinely.